constants/pools.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "pool": "vex-vet",
+    "address": "0x538f8890a383c44e59df4c7263d96ca8048da2c7",
+    "monthlyRate": 387460
+  },
+  {
+    "pool": "wov-vet",
+    "address": "0xa8d1a1c88329320234581e203474fe19b99473d3",
+    "monthlyRate": 3000
+  },
+  {
+    "pool": "veed-vet",
+    "address": "0x40b0972f74d44da2b7a5a5817f81d38e293bce9f",
+    "monthlyRate": 5000
+  },
+  {
+    "pool": "veusd-vet",
+    "address": "0xf1be58861b4bcacd6c7d026ba3de994361f5d3aa",
+    "monthlyRate": 5000
+  }
+]
```

Comment:
let's include all the (new) core pairs we're planning to launch on 1 May as well